Users.py

```
@@ -0,0 +1,143 @@
+    )
+
+
+@app.post("/auth/login",response_model=UserResponse)
+async def login(login_data : Annotated[UsersBase, Depends(UsersBase.as_form)]):
+    users = get_db_users()
+    user_record = next((u for u in users if u["email"] == login_data.email), None)
+    token, expires_in = create_access_token(user_record["user_id"])
+
+    return UserResponse(
+        access_token=token,
+        token_type="bearer",
```

Ruff (0.14.11)

Possible hardcoded password assigned to argument: "token_type"

(S106)